src/particles/spacecharge/Gauss3dPush.cpp

```diff
@@ -0,0 +1,202 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Ji Qiang
+ * License: BSD-3-Clause-LBNL
+ */
+#include "Gauss3dPush.H"
+
+#include <AMReX_BLProfiler.H>
+#include <AMReX_REAL.H>       // for Real
+#include <AMReX_SPACE.H>      // for AMREX_D_DECL
+#include "diagnostics/ReducedBeamCharacteristics.H"
+
+
+namespace impactx::particles::spacecharge
+{
+    /** Compute space-charge fields from a 3D Gaussian distribution.
+     *
+     * Compute integrals Eqs 45-47 used in the space-charge fields from a 3D Gaussian distribution.
+     * "A two-and-a-half dimensional symplectic space-charge solver", LBNL Report Number: LBNL-2001674, 2025.
+     * Input particle locations (x,y,z) and RMS sizes (sigx,sigy,sigz) and return the integrals for SC fields.
+     *
+     * @todo This function can be vectorized with AMREX_SIMD for better CPU performance.
+     */
```

Comment:
Ooops, this block of the doc string should be moved to the `.H` file doc string (just append to current content).

Reply:
No, this is defined inline in this file, all good.